pkg/model/const.go

```diff
@@ -277,6 +277,18 @@ const (
 	// OktetoDivertIstioDriver is the divert driver for istio
 	OktetoDivertIstioDriver = "istio"
 
-	// OktetoDivertHeader the header used by okteto to divert traffic
-	OktetoDivertHeader = "x-okteto-divert"
+	// OktetoDivertIstioExactMatch represents the exact header match type
+	OktetoDivertIstioExactMatch = "exact"
```

Comment:
Can we moved the constants related to the divert implementation to `pkg/constants`. We want to stop growing the model package